airbyte-integrations/connectors/source-iterable/source_iterable/streams.py

```diff
@@ -354,11 +354,29 @@ class ListUsers(IterableStream):
     data_field = "getUsers"
     name = "list_users"
     # enable caching, because this stream used by other ones
-    use_cache = True
+    use_cache = False
```

Comment:
shouldn't we leave it as cacheable stream?

Reply:
In this case {stream: True} has no any effect

Reply:
but don't we use List Users stream as parent for Events ?